BinaryEncoding.md

```diff
@@ -74,7 +74,7 @@ The following documents the current prototype format. This format is based on an
 
 ## High-level structure
 
-The module starts with a magic number and version as follows.
+A WASM binary is comprised of a single module and optionally data such as function names and data segment contents trailing the module. The module starts with a magic number and version as follows.
```

Comment:
I think we can simplify this text to: "A WASM binary contains a single module, which starts with a magic number and version number as follows". The rest of the document describes the rest of the contents in enough detail that I think the summary here isn't necessary.